curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md

```diff
@@ -0,0 +1,418 @@
+
+```js
+const el = new __helpers.CSSHelp(document).getStyle('body').getPropVal('font-family');
+console.log(el);
+const condition1= el === 'Arial, sans-serif';
+const condition2= el === '"Arial", sans-serif';
+assert(condition1 || condition2);
+```
+
+
+Your `p` elements should have a top and bottom margin of `5px`.
+
+```js
+const margin = new __helpers.CSSHelp(document).getStyle('p').getPropVal('margin');
+const marginTop = new __helpers.CSSHelp(document).getStyle('p').getPropVal('margin-top');
+const marginBottom = new __helpers.CSSHelp(document).getStyle('p').getPropVal('margin-bottom');
+
+const condition1 = margin === '5px 0';
+const condition2 = marginTop === '5px' && marginBottom === '5px';
+const condition3 = margin === '5px 0 5px 0';
+assert(condition1 || condition2 || condition3);
+```
+
+The links of the page should have no underline by default.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('a').getPropVal('text-decoration'), 'none');
+```
+
+The page links should have an underline when hovered over.
+
+```js
+const gs = (s) => new __helpers.CSSHelp(document).getStyle(s);
+const a_hover = gs('a:hover')?.textDecoration;
+assert.equal(a_hover, 'underline');
+```
+
+You should have one main `div` with the class `business-card`.
+
+```js
+assert(document.querySelector('div')?.getAttribute('class') === 'business-card');
+```
+
+Your `.business-card` class should have a width of `300px`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('width'), '300px');
+```
+
+Your `.business-card` class should have a background color of your choice.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.business-card')?.backgroundColor);
+```
+
+Your `.business-card` class should have a padding of `20px`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('padding'), '20px');
+```
+
+Your `.business-card` class should have a margin of `100px` from the top.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('margin-top'), '100px');
+```
+
+The text inside the `.business-card` class should be center-aligned.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('text-align'), 'center');
+```
+
+The font size of the text inside the `.business-card` class should be `16px`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('font-size'), '16px');
+```
+
+The left and right margins of the `.business-card` class should be set to `auto`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.business-card').getPropVal('margin-left'), 'auto');
+```
+
+Inside the `.business-card` class, there should be an `img` element with the class `profile-image`.
+
+```js
+assert.exists(document.querySelector('img.profile-image'));
+```
+
+The image source should be set to a valid image.
```

Comment:
Once `img.profile-image`  is created this is always satisfied, even if you don't set the `src`.
I would check the length of the attribute instead.

Reply:
It should work now.